.github/workflows/test-template.yml

```diff
@@ -59,7 +59,7 @@ jobs:
   #
   lint:
     name: Lint
-    timeout-minutes: 7
```

Comment:
Does our lint take that long? 🫣

Reply:
Yeah, just the build itself now takes that long, it doesn't even get to the linting 😅